### PR TITLE
Improve scrollbar grab width on hidpi devices

### PR DESCRIPTION
**Pull request by harshad1**
Currently, the draggable scrollbar grab width uses the raw pixel width of the scoll bar. On higher DPI devices (which is most devices these days) the grab region can be too small.

For example, the default on my pixel 3 XL is 0.7mm.

This PR uses a better heuristic which accounts for DPI

### CODE REVIEW

**Comment by harshad1** [comment]
@gsantner `DraggableScrollbarScrollView.java` and `DraggableScrollbarWebView.java` are 95% identical. We should probably find a way to de-duplicate them,

File: app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java [hunk 1]
```diff
@@ -80,7 +81,8 @@ protected void onAttachedToWindow() {
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN_MR1) {
             _ltr = getLayoutDirection() == View.LAYOUT_DIRECTION_LTR;
         }
-        _scrollbarWidth = getVerticalScrollbarWidth();
+        final DisplayMetrics displayMetrics = getContext().getResources().getDisplayMetrics();
+        _grabWidth = (int) (2.0 * (float) getVerticalScrollbarWidth() * displayMetrics.density);
```
Comment (harshad1) [inline app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java:85]: @gsantner It would be good to test this on other devices

Comment (harshad1) [reply app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java:85]: I have tested this in the emulator and it seems ok

Comment (gsantner) [reply app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java:85]: working here as well

**Review by gsantner — APPROVED** [review]

**Comment by gsantner** [comment]
thanks for so much help and your time. Everything merged. Excpet the last used format, not sure yet if its all fine.

**Comment by harshad1** [comment]
> thanks for so much help and your time. Everything merged. Except the last used format, not sure yet if its all fine.

No problem. 99% of my PRs are things that I want or that directly affect my usage, win-win. Last used format should be ready to merge too

**Pull request merged**